ThunkLibs/libX11/libX11_interface.cpp

```diff
@@ -636,3 +636,48 @@ template<> struct fex_gen_config<_XInitImageFuncPtrs> : fexgen::custom_guest_ent
 template<> struct fex_gen_config<XVaCreateNestedList> {
     using uniform_va_type = void*;
 };
+
+// Xlibint
+template<> struct fex_gen_config<_XError> {};
+template<> struct fex_gen_config<_XIOError> {};
+template<> struct fex_gen_config<_XGetHostname> {};
+template<> struct fex_gen_config<_XScreenOfWindow> {};
+template<> struct fex_gen_config<_XAsyncErrorHandler> {};
+template<> struct fex_gen_config<_XGetAsyncData> {};
+template<> struct fex_gen_config<_XEventsQueued> {};
+template<> struct fex_gen_config<_XEnq> {};
+template<> struct fex_gen_config<_XDeq> {};
+template<> struct fex_gen_config<_XUnknownWireEvent> {};
+template<> struct fex_gen_config<_XUnknownWireEventCookie> {};
+template<> struct fex_gen_config<_XUnknownCopyEventCookie> {};
+template<> struct fex_gen_config<_XWireToEvent> {};
+template<> struct fex_gen_config<_XDefaultWireError> {};
+template<> struct fex_gen_config<_XPollfdCacheInit> {};
+template<> struct fex_gen_config<_XPollfdCacheAdd> {};
+template<> struct fex_gen_config<_XPollfdCacheDel> {};
+template<> struct fex_gen_config<_XAllocID> {};
+template<> struct fex_gen_config<_XAllocIDs> {};
+template<> struct fex_gen_config<_XFreeExtData> {};
+// XXX: Needs investigation in to this API
```

Comment:
What kind of investigation is needed that wasn't needed for the other functions?

Reply:
Oops, that was just a note to myself. Just needed to check that it matched the other `XESet*` functions and they do.